cmd/evtest/main.go

```diff
@@ -34,6 +34,35 @@ func listDevices() {
 	}
 }
 
+func cloneDevice(dev *evdev.InputDevice) (*evdev.InputDevice, error) {
+	return evdev.CloneDevice("my-device-clone", dev)
+}
+
+func createDevice() (*evdev.InputDevice, error) {
```

Comment:
Why is this still in `evtest`? Now that you added a new utility, you can just remove these functions, right?

Reply:
removed